src/Stage/Parse/Parser.elm

```diff
@@ -835,6 +1065,68 @@ newlines =
     P.chompWhile ((==) '\n')
 
 
+checkIndent : (Int -> Int -> Bool) -> ParseProblem -> Parser_ ()
+checkIndent check error =
+    P.succeed
+        (\indent col ->
+            if check indent col then
+                P.succeed ()
+
+            else
+                P.problem error
+        )
+        |= P.getIndent
+        |= P.getCol
+        |> P.andThen identity
+
+
+ignorablesAndCheckIndent : (Int -> Int -> Bool) -> ParseProblem -> Parser_ ()
+ignorablesAndCheckIndent check error =
```

Comment:
Similar to the above comment for `checkIndent` - and please tell us how does it differ from it (preferably with examples). Thanks!